### PR TITLE
✨[FEAT]: Manage Synchronized Instances (Step 1  + Auto-remove/Unsecured error)

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [x] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

### 📝 Description

Manage Synchronized Instances (Step 1)

https://github.com/user-attachments/assets/45253fdb-8ab7-41bb-9df5-c554e029d640

Auto-remove/Unsecured error 

https://github.com/user-attachments/assets/778a52cd-5b67-4fde-a9ec-e0a9014993a8

### ❓ Context

- **JIRA or GitHub link**: [LIVE-12159] 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-12159]: https://ledgerhq.atlassian.net/browse/LIVE-12159?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/__integrations__/manageInstances.integration.test.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+      }),
+    });
+
+    // Check if the ledger sync row is visible
+    await expect(await screen.findByText(/ledger sync/i)).toBeVisible();
+
+    // On Press the ledger sync row
+    await user.press(await screen.findByText(/ledger sync/i));
+
+    //Manga Key Flow
```
Comment (LucasWerey) [inline apps/ledger-live-mobile/src/newArch/features/WalletSync/__integrations__/manageInstances.integration.test.tsx:61]:
```suggestion
    //Manage Key Flow
```

File: .changeset/ninety-seas-travel.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"live-mobile": patch
+---
+
+Mange Key flow
```
Comment (LucasWerey) [inline .changeset/ninety-seas-travel.md:5]: Manage

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/__integrations__/manageKey.integration.test.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+      }),
+    });
+
+    // Check if the ledger sync row is visible
+    await expect(await screen.findByText(/ledger sync/i)).toBeVisible();
+
+    // On Press the ledger sync row
+    await user.press(await screen.findByText(/ledger sync/i));
+
+    //Manga Key Flow
```
Comment (LucasWerey) [inline apps/ledger-live-mobile/src/newArch/features/WalletSync/__integrations__/manageKey.integration.test.tsx:48]: manage

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/ManageInstances/ManageInstancesDrawer.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+export enum Scene {
+  List,
+  Instructions,
+  AutoRemove,
+  Unsecured,
+}
+
+const ManageInstancesDrawer = ({ isOpen, handleClose }: Props) => {
+  const { isError, error, isLoading, data } = useGetMembers();
+  const memberCredentials = useSelector(memberCredentialsSelector);
```
Comment (KVNLS) [inline apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/ManageInstances/ManageInstancesDrawer.tsx:28]: We should not rely on redux Store inside the view. Could you move it in the viewModel hook, please?

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/ManageInstances/useManageInstanceDrawer.ts [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import { useState, useCallback } from "react";
```
Comment (KVNLS) [inline apps/ledger-live-mobile/src/newArch/features/WalletSync/screens/ManageInstances/useManageInstanceDrawer.ts:1]: If this hook is not a viewModel hook, we should put it inside the hooks folder

File: apps/ledger-live-mobile/src/newArch/features/WalletSync/hooks/useWalletSyncAnalytics.ts [hunk 6]
```diff
@@ -41,6 +42,8 @@ export enum AnalyticsButton {
   Cancel = "Cancel",
   CreateYourKey = "Create your key",
   LedgerSync = "Ledger Sync",
+  UseAnother = "connect new ledger",
```
Comment (KVNLS) [inline apps/ledger-live-mobile/src/newArch/features/WalletSync/hooks/useWalletSyncAnalytics.ts:45]: Don't we want a capital letter for `connect`?

**Review by KVNLS — CHANGES_REQUESTED** [review]

**Pull request merged**